client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -160,22 +186,27 @@ void main(void)
 	color = base.rgb;
 #endif
 
+	vec4 col = vec4(color.rgb, 1.0); 
```

Comment:
can you merge this calcul with the next line to limit calculs and permit GPU to optimize this calcul ?